client/reader/user-profile/views/recommended-blogs.tsx

```diff
@@ -48,12 +50,21 @@ const UserRecommendedBlogs = ( { user }: UserRecommendedBlogsProps ): JSX.Elemen
 	}
 
 	if ( ! recommendedBlogs?.length ) {
+		const action = currentUser?.username === userLogin && (
+			<a
+				className="empty-content__action button is-primary"
+				href={ `/reader/list/${ userLogin }/recommended-blogs/edit/items` }
+			>
+				{ translate( 'Add recommendations' ) }
```

Comment:
Should it be "Add recommended blogs" or is that too long? We could request editorial review if that would help.

Reply:
I don't have a strong opinion. 

> We could request editorial review if that would help.

I put in my best first guess for copies and DM'd @davemart-in the images for copy feedback earlier:

![Screenshot 2025-07-09 at 1 44 49 PM](https://github.com/user-attachments/assets/612c7a7a-5cdc-4eef-9007-e8f65bac957d)

Im happy to make any updates that make sense, but don't have a strong opinion on them.